src/main/java/cloud/fogbow/ras/core/plugins/interoperability/aws/network/v2/AwsV2NetworkPlugin.java

```diff
@@ -187,17 +190,14 @@ private void handleSecurityIssues(String cidr, String subnetId, Ec2Client client
 			CreateSecurityGroupResponse response = client.createSecurityGroup(request);
 			groupId = response.groupId();
 			doCreateTagsRequests(AWS_TAG_GROUP_ID, groupId, subnetId, client);
-			doAuthorizeSecurityGroupIngressRequests(cidr, subnetId, groupId, client);
+			doAuthorizeSecurityGroupIngress(cidr, subnetId, groupId, client);
 		} catch (Exception e) {
```

Comment:
can it be SdkException?

Reply:
I thought of something more generic, thinking in the case of a return on "response.groupId ()", but the AWS cloud should not return this if the request succeeds. Then I will accept your suggestion.